Home.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>conference page(home)</title>
+    <script src="https://kit.fontawesome.com/fe8431d094.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://use.typekit.net/ubw2tuq.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+  <div class="upper">
```

Comment:
- [ ] For semantic reasons, kindly make this a `nav` tag instead of a `div` tag.